packages/app-store/applecalendar/lib/CalendarService.ts

```diff
@@ -1,8 +1,8 @@
 import CalendarService from "@calcom/lib/CalendarService";
-import type { CredentialPayload } from "@calcom/types/Credential";
+import type { CredentialWithAppName } from "@calcom/types/Credential";
```

Comment:
Changing to type `CrednetialWithAppName` because the email is required but the id was always optional.